src/azure-cli/azure/cli/command_modules/redis/_params.py

```diff
@@ -42,6 +42,7 @@ def load_arguments(self, _):
         c.argument('vm_size', arg_type=get_enum_type(allowed_c_family_sizes + allowed_p_family_sizes), help='Size of Redis cache to deploy. Basic and Standard Cache sizes start with C. Premium Cache sizes start with P')
         c.argument('enable_non_ssl_port', action='store_true', help='If specified, then the non-ssl redis server port (6379) will be enabled.')
         c.argument('replicas_per_master', help='The number of replicas to be created per master.', is_preview=True)
+        c.argument('ports', type=validate_list_of_integers, help='Specifies the ports of nodes to be rebooted. Comma separated list of integers.', is_preview=True)
```

Comment:
Please use the first person voice to write the help
```suggestion
        c.argument('ports', type=validate_list_of_integers, help='Specify the ports of nodes to be rebooted. Comma separated list of integers.', is_preview=True)
```